test/integration/objectmeta/objectmeta_test.go

```diff
@@ -19,70 +19,36 @@ package objectmeta
 import (
 	"testing"
 
-	etcd "github.com/coreos/etcd/client"
-	"github.com/golang/glog"
 	"github.com/stretchr/testify/assert"
-	"golang.org/x/net/context"
-	"k8s.io/kubernetes/pkg/api/testapi"
+
```

Comment:
Can we limit this PR to change integration tests?
Are we already using etcd v3 API there? If not, can you also switch it?